govcd/cse_internal_unit_test.go

```diff
@@ -0,0 +1,235 @@
+//go:build unit || ALL
+
+package govcd
+
+import (
+	semver "github.com/hashicorp/go-version"
+	"os"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+// Test_cseClusterSettingsInternal_generateCapiYamlAsJsonString
+func Test_cseClusterSettingsInternal_generateCapiYamlAsJsonString(t *testing.T) {
```

Comment:
Awesome test! Please add a comment describing more about it.

Reply:
Done, thanks